gadgets/profile_tcprtt/README.mdx

```diff
@@ -0,0 +1,290 @@
+
+        ```bash
+        # Run the gadget again
+        $ kubectl gadget run profile_tcprtt:latest --node minikube-docker
+        latency
+                µs               : count    distribution
+                 0 -> 1          : 0        |                                        |
+                 1 -> 2          : 0        |                                        |
+                 2 -> 4          : 0        |                                        |
+                 4 -> 8          : 6        |********                                |
+                 8 -> 16         : 5        |******                                  |
+                16 -> 32         : 15       |********************                    |
+                32 -> 64         : 9        |************                            |
+                64 -> 128        : 15       |********************                    |
+               128 -> 256        : 0        |                                        |
+               256 -> 512        : 0        |                                        |
+               512 -> 1024       : 0        |                                        |
+              1024 -> 2048       : 0        |                                        |
+              2048 -> 4096       : 4        |*****                                   |
+              4096 -> 8192       : 30       |****************************************|
+              8192 -> 16384      : 19       |*************************               |
+             16384 -> 32768      : 0        |                                        |
+             32768 -> 65536      : 0        |                                        |
+             65536 -> 131072     : 0        |                                        |
+            131072 -> 262144     : 0        |                                        |
+            262144 -> 524288     : 0        |                                        |
+            524288 -> 1048576    : 0        |                                        |
+           1048576 -> 2097152    : 0        |                                        |
+           2097152 -> 4194304    : 0        |                                        |
+           4194304 -> 8388608    : 0        |                                        |
+           8388608 -> 16777216   : 0        |                                        |
+          16777216 -> 33554432   : 0        |                                        |
+          33554432 -> 67108864   : 0        |                                        |
+        ```
+
+        Now, let's use the [network emulator](https://wiki.linuxfoundation.org/networking/netem) to introduce some random delay to the packets and increase indirectly the RTT:
+
+        ```bash
+        # tc qdisc add dev eth0 root netem delay 50ms 50ms 25%
+        # curl nginx
+        # curl nginx
+        ```
+
+        Now the average RTT value of the new histogram increased:
```

Comment:
I wasn't able to get the same result on my machine as the gadget isn't filtering by container, so the noise of all operations hides the delay effect on this specific container.

Trying to download a bigger file (like ` wget https://testfile.org/1.3GBiconpng` helped)